ci/docs/layout.html

```diff
@@ -12,4 +12,36 @@
   ga('send', 'pageview');
 
 </script>
+
+{# check current version against supported-releases.json #}
+<script>
+$( document ).ready( function() {
+
+  // gets version number minus whitespace and bugfix version
+  var $version_number = $('.version').text().trim().split('.', 2).join('.')
+
+  var supported_versions = []
+  // get JSON from server and load supported_versions
+  $.getJSON("https://raw.githubusercontent.com/pulp/pulp_packaging/master/ci/config/releases/supported-releases.json" , function(data) {
```

Comment:
This is actually my first time using ajax. Would the following be valid?


`var ajax = $.getJSON( //stuff)`
`ajax.done(function() { //other stuff }`